dom/observable/tentative/observable-finally.any.js

```diff
@@ -0,0 +1,277 @@
+test(() => {
+  const source = new Observable((subscriber) => {
+    subscriber.next(1);
+    subscriber.next(2);
+    subscriber.next(3);
+    subscriber.complete();
+  });
+
+  const results = [];
+
+  source
+    .finally(() => {
+      results.push("finally called");
+    })
+    .subscribe({
+      next: (value) => results.push(value),
+      error: (e) => results.push(e.message),
+      complete: () => results.push("complete"),
+    });
+
+  assert_array_equals(
+    results,
+    [1, 2, 3, "complete", "finally called"],
```

Comment:
```suggestion
    [1, 2, 3, "finally called", "complete"],
```

This one was incorrect. Sorry.